jte-gradle-plugin/src/main/java/gg/jte/gradle/JteGradle.java

```diff
@@ -0,0 +1,12 @@
+package gg.jte.gradle;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+
+public class JteGradle implements Plugin<Project> {
+    @Override
+    public void apply(Project project) {
+        project.getTasks().register("generatorMojo", GeneratorMojoTask.class);
```

Comment:
Mojo is coming from maven. I'm not very proficient in gradle, so I'm not sure if there's something similar there. If not, we might just call it `generator`?

Hmm, or since it is named task, maybe just `generate` and `compile`?

Reply:
It's a task name, from gradle we can access it via `tasks.name`, like `tasks.generatorMojo`, I think it's not a best idea to give a task such a general name. I've looked at the source code of the maven plugin, now I think it was, uh, bad. Don't you think `generateJte` & `compileJte` would be better? I guess the task name should follow the package name, but I'm not sure tbh.

Reply:
Agreed! I renamed the tasks as you suggested: https://github.com/casid/jte/commit/82ba9d93d3c59fd75b2a6702c1aec251df3ed1a4